.gitignore

```diff
@@ -3,6 +3,7 @@
 
 /bin/
 /vendor/
+.idea
```

Comment:
Please read the comment on top of the file

Reply:
Sorry. The line has been removed